source/_integrations/proximity.markdown

```diff
@@ -43,10 +42,24 @@ For each tracked device or person, a sensor is created showing the direction of
 - `towards`
 - `unknown`
 
+<div class="note">
+To calculate the distance and the direction of travel for a tracked device or person, they must specify a geo-location.
+</div>
+
 ### Nearest device
 
 A sensor is created showing the device or person which is nearest (_shortest distance_) to the monitored zone. If several devices or persons are at the same nearest distance, this sensor displays them all.
 
-<div class="note">
-To calculate the distance and the direction of travel for a tracked device or person, they must specify a geo-location.
-</div>
+### Nearest Distance
```

Comment:
```suggestion
### Nearest distance
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headings